wire-ios-system/Source/ZMSLog.swift

```diff
@@ -293,6 +309,12 @@ extension ZMSLog {
         return manager.urls(for: .cachesDirectory, in: .userDomainMask).first
     }
 
+
+    static var tmpDirectory: URL? {
+        let tempUrl = URL(string: NSTemporaryDirectory())
+        return tempUrl
+    }
```

Comment:
Where was the temp directory previous written to? It seems to be a new addition to this file so I'm a little confused how logs are written to the temp directory.

Reply:
yes confused with log cache and temp cache. Moved from zmlogs